Gemfile

```diff
@@ -71,5 +71,6 @@ group :test do
   gem "webdrivers"
 end
 
```

Comment:
![image](https://user-images.githubusercontent.com/93039359/224970893-de6ed8ff-5a72-4dbb-8fc9-bc74bffcbb9a.png)

- [ ] It would be nice if you replace the `npm gemfile` with `bundle install` command because `bundle install` can install the gems from your gemfile :blush: